community/openwhisk/values.yaml

```diff
@@ -24,7 +24,7 @@ whisk:
   # See docs/ingress.md for a discussion of how to provide these values.
   ingress:
     name: "ow-ingress"
-    apiHostName: ""
+    apiHostName: 9.46.68.68
```

Comment:
don't think we should be adding this host, shouldn't this be left blank?

Reply:
yes, I have fixed this